tst/testbugfix/2018-05-24-IntermediateSubgroups.tst

```diff
@@ -0,0 +1,24 @@
+# test for MaximalSubgroupClassReps with options (reported through
+# observation by S.Alavi with IntermediateGroup
+# More complicated to construct w/o AtlasSubgroup to ensure the 325 points action, as it is too
+# slow otherwise. Also construct the smaller subgroup s1 directly.
+# Finally do not slow down with assertions that don't need testing here
+
+gap> SetAssertionLevel(0);;
```

Comment:
If you turn off assertions here, this will also turn them off in subsequent tests -- until some test uses `START_TEST/STOP_TEST`, which not all .tst files do right now.
So please either manually save/restore the assertion level, or else (probably simpler) add `START_TEST` and `STOP_TEST` calls.

Alternatively, somebody else can add this after this PR gets merged.